main.js

```diff
@@ -0,0 +1,247 @@
+const display = document.getElementById("display");
+const start = document.querySelector(".start");
+const input = document.querySelector("#input");
+document.querySelector("button").onclick = myClick;
+
+display.innerHTML = "00:00";
+
+let timeOut;
+let time = myClick;
+function myClick() {

```

Comment:
There is no validation for entered values, timer works even with empty values


https://user-images.githubusercontent.com/84010501/200114576-69bd2a5f-6db9-42a6-a7b7-b156f175e744.mov